www/attributes/hx-push-url.md

```diff
@@ -7,7 +7,7 @@ title: </> htmx - hx-push-url
 
 The `hx-push-url` attribute allows you to "push" a new entry into the browser location bar, which creates
 a new history entry, allowing back-button and general history navigation.  The possible values of this
-attribute are `true` and `false`.
+attribute are `true`, `false` and a custom string.
```

Comment:
Could you change this to "or a custom string", I think the original sentence was incorrect. :/

Reply:
Agreed and done.